CHANGELOG.md

```diff
@@ -2701,21 +2847,13 @@ This section is only relevant to contributors.
 
 [unreleased]: https://github.com/deltachat/deltachat-desktop/compare/v1.46.1...HEAD
 
-[1.46.1]: https://github.com/deltachat/deltachat-desktop/compare/v1.46.0...v1.46.1
 
-[1.45.5]: https://github.com/deltachat/deltachat-desktop/compare/v1.45.5...v1.46.0
```

Comment:
Links need to be fixed instead. Plus it'd be better if it was a separate commit, better yet a separate MR.

Reply:
I didn't. LazyVim did xD